CMakeLists.txt

```diff
@@ -497,6 +497,25 @@ if(CELERITAS_BUILD_TESTS)
   celeritas_find_or_builtin_package(GTest 1.10)
 endif()
 
+#----------------------------------------------------------------------------#
+# System property: availability of sincospi
+if(NOT DEFINED CELERITAS_SINCOSPI_PREFIX)
```

Comment:
For clarify I would have set the full name of the function instead of just the prefix.  (Albeit technically the existing code could be abstract out to test/check another symbol with the same pattern).  Either way minor issue.

Reply:
It's not just one function though, it's several... #1364